### PR TITLE
MAT-9442: update translator-commons

### PR DESCRIPTION
**Pull request by chubert-sb**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-9442](https://jira.cms.gov/browse/MAT-9442)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -105,38 +105,38 @@
...
-    <dependency>
-      <groupId>info.cqframework</groupId>
-      <artifactId>elm-jaxb</artifactId>
-      <version>${cqframework.version}</version>
-    </dependency>
+    <!--    <dependency>-->
+    <!--      <groupId>org.cqframework</groupId>-->
+    <!--      <artifactId>elm-jaxb</artifactId>-->
+    <!--      <version>${cqframework.version}</version>-->
+    <!--    </dependency>-->
```
Comment (jkotanchik-SB) [inline pom.xml]: No love for jaxb?

File: pom.xml [hunk 2]
```diff
@@ -105,38 +105,38 @@
...
-    <dependency>
-      <groupId>info.cqframework</groupId>
-      <artifactId>model-jaxb</artifactId>
-      <version>${cqframework.version}</version>
-    </dependency>
+    <!--    <dependency>-->
+    <!--      <groupId>org.cqframework</groupId>-->
+    <!--      <artifactId>model-jaxb</artifactId>-->
+    <!--      <version>${cqframework.version}</version>-->
+    <!--    </dependency>-->
```
Comment (jkotanchik-SB) [inline pom.xml]: No love for jaxb?

File: src/test/java/gov/cms/mat/cql_elm_translation/service/CqlConversionServicePropertyTest.java [hunk 3]
```diff
@@ -90,9 +90,10 @@ void process_annotations() {
 
   @Test
   void process_locators() {
-    String locatorTag = "\"locator\" : ";
+    String locatorTag = "\"locator\":";
 
     String jsonDefault = getJson();
+    System.out.println(jsonDefault);
```
Comment (jkotanchik-SB) [inline src/test/java/gov/cms/mat/cql_elm_translation/service/CqlConversionServicePropertyTest.java]: Can this debugging statement be removed?

File: pom.xml (hunk 1 above)
Comment (chubert-sb) [reply pom.xml]: forgot to clean that up

**Review by jkotanchik-SB — DISMISSED** [review]

File: pom.xml (hunk 2 above)
Comment (chubert-sb) [reply pom.xml]: no love

File: src/test/java/gov/cms/mat/cql_elm_translation/service/CqlConversionServicePropertyTest.java (hunk 3 above)
Comment (chubert-sb) [reply src/test/java/gov/cms/mat/cql_elm_translation/service/CqlConversionServicePropertyTest.java]: removed

**Review by jkotanchik-SB — APPROVED** [review]
